src/Client/widgets/Task.fs

```diff
@@ -10,157 +10,226 @@ open Fable.FontAwesome
 open Fable.FontAwesome.Free
 open Fulma
 open Markup
+open Utils
+
```

Comment:
As for the whole `Task` module - mostly I get the changes here, I have two general notes:
1) Sometimes we use `btn` and sometimes we use `button` for button - ideally we should have same names for the same things/concepts
2) The whole module is sort of becoming fat... Do you think there is an easy and clear way to split it somehow?

Reply:
I have made some initial split for that, if we'll have to make any additions to the module it will be easier to split then. For now I don't want to make it more complicated

Reply:
Alright. Then just probably keep in mind that I become nervous when a module starts to exceed 100 LOC :D